types/TestRunner.js

```diff
@@ -18,14 +18,16 @@ import type {
   TestResult,
 } from 'types/TestResult';
 import type Runtime from 'jest-runtime';
-import type TestWatcher from 'jest-cli/src/test_watcher';
+import type {TestWatcher as _TestWatcher} from 'jest-cli';
```

Comment:
It's required in this file to be exported under the same name, that way I get around making jest-runner depend on jest-cli, which would be a circular dependency just to include the type information. `export type TestWatcher = TestWatcher` doesn't work in Flow.